website/project/model.py

```diff
@@ -2749,6 +2749,8 @@ def embargo_registration(self, user, end_date, for_existing_registration=False):
         )
         # Embargo record needs to be saved to ensure the forward reference Node->Embargo
         self.embargo = embargo
+        if self.is_public:
+            self.set_privacy('private', Auth(user))
```

Comment:
@sloria good point. It will but I'm not sure it's needed as it's implied that an embargo should be private. Thoughts?